test/unit/validate.test.js

```diff
@@ -0,0 +1,27 @@
+import assert from 'assert';
+import validate from '../../src/validate.js';
+
+describe('validate', () => {
+  test('validate uuid', () => {
+    assert.strictEqual(validate('00000000-0000-0000-0000-000000000000'), true);
```

Comment:
Great that you extracted these tests as suggested by @broofa !

However, I believe we should keep _one_ test with an invalid namespace in the `v3` and `v5` test suite in order to ensure that the error thrown by `validate()` is in fact propagated and not somehow swallowed by the uuid generation methods.

Reply:
@ctavan It is already there:
https://github.com/uuidjs/uuid/blob/d5ce415363/test/unit/v35.test.js#L99-L109
https://github.com/uuidjs/uuid/blob/d5ce415363/test/unit/v35.test.js#L195-L205